src/components/Layout/Seo.tsx

```diff
@@ -47,6 +47,10 @@ const SEO: React.FC<ISEO> = (props) => {
             `${site.siteMetadata.siteUrl}${useLocation()?.pathname}?lang=en`,
           rel:'canonical',
         },
+        {
+          href: '/rss.xml',
+          rel: 'alternate',
```

Comment:
Please add
```
title: 'RSS Feed for blog.maxmind.com`

Reply:
Sure, I'll add it in. I tried adding it in earlier, but I didn't see it actually affect the `<link rel="alternate" type="application/rss+xml" title="RSS Feed for blog.maxmind.com" href="/rss.xml" />`. In other words, when I did, it was showing up as `<link rel="alternate" type="application/rss+xml" title="MaxMind Blog" href="/rss.xml" />` because I _thought_ it was pulling the title from below.

It seemed like the title was coming from `index.ts` file 
```
},
            title: 'RSS Feed for blog.maxmind.com',
          },
        ],
        query: `
          {
            site {
              siteMetadata {
                title
                description
                siteUrl
                site_url: siteUrl
              }
            }
          }
        `,
      },
      resolve: 'gatsby-plugin-feed',
```